RLTSquare/CustomEmailTemplate/README.md

```diff
@@ -0,0 +1,35 @@
+# RLTSquare_CustomEmailTemplate module
+
+<font color='red'>**The following example is a complete README for a module RLTSquare_CustomEmailTemplate:** </font>
+# RLTSquare_CustomEmailTemplate module
+The RLTSquare_CustomEmailTemplate module enables you to add the Configurable Product updates to the existing store campaigns.
+
+The RLTSquare_CustomEmailTemplate module is a part of the staging functionality in Magento EE. The module adds the “Configurations” tab and the configuration wizard to the Schedule Update form of a product. You can change the Configurable Product attributes in campaigns. These updates are shown on the campaign dashboard.
+
+## Installation details
+
+The RLTSquare_CustomEmailTemplate module makes irreversible changes in a database during installation. You cannot disable or uninstall this module.
```

Comment:
Overcomplicated code - as far as I can see, module has no install / patch scripts, so the only "change" is the fact it gets installed and registered in Magento ... this is all default ... so I would not emphasize it in documentation in any such way.